handler/pullRequestHandler.go

```diff
@@ -143,3 +179,7 @@ func hasUnsigned(req types.PullRequestOuter, client *github.Client) (bool, error
 func isSigned(msg string) bool {
 	return strings.Contains(msg, "Signed-off-by:")
 }
+
+func missingDescription(pr types.PullRequest) bool {
+	return len(pr.Body) < 1
```

Comment:
Not sure this could ever be negative 1 `-1` or less than `0` so how about looking at `== 0`.

One other thought, what if someone pasted this:

```
 
```

(that's a single space)

This may be of use for extending the tests:

```
strings.Trim*
```

Reply:
The `< 1` is my personal standard for testing string length since some languages return a -1 when the value cannot be determined. I know that's not the case in Go, just a habit of mine.

For the blank space, I did test this with Github, and found that they trim pre and post spaces before the request is sent, so I didn't add the additional step. It wouldn't hurt, so I'll add that in just in case.

Reply:
Not sure I follow.
Are you saying this check should validate that there is a word in the description? Rather than just testing the length.

Reply:
I was thinking that by using a regex we could omit whitespace.  On reflection there will be all manner of non-alphanumerics as its markdown, so let's consign that suggestion to the heap.

Reply:
What can be done is to check the commit body too, as it's usually automatically added as description.
Three problems with that are
* Still does not guarantee non-empty description
* Moves the scope of the check
* Complicates for multiple commits

But it's still an option.

Reply:
The scope of this issue is only for non-empty bodies. Please see the issue description.

Reply:
Ok, I just thought for the case, when the body is not empty and still there is no description. But this can be out of scope for this PR.

Reply:
I agree, it's encapsulated in another issue - this one has a much more defined scope.

Reply:
I've updated this piece with the input provided. Added a trim, and check for 0.